### ISSUE TITLE
Combiner taking too much time to execute

### ISSUE THREAD

**Issue by FahadShahid143** [amphibian-dev/toad#131]
Combiner takes too much time to execute. Below code takes several hours to execute. Is there a way to improve it? 

combiner = toad.transform.Combiner()
combiner.fit(X=x_train,
             y=x_train['label'],
             method='chi',
             min_samples = 0.05,
             exclude=exclude_list)

**Comment by Secbone** [comment]
@FahadShahid143 yes, it does only merge two values in a single step when using 'chi', it will take a long time when the unique value is too much, so I think if it can merge multiple values in a single step will improve the speed.
I will work on it, and release the new version as soon as possible

**Comment by FahadShahid143** [comment]
@Secbone is there any update on this? Did you get a chance to look into it?
